client/web/src/components/widgets/PopoverMount.vue

```diff
@@ -91,5 +94,88 @@ export default defineComponent({
 			if (bottomOffset < 0) popoverContent.style.top = `${bottomOffset}px`;
 		}
 	},
+	methods: {
+		setOpen() {
+			this.open = true;
+		},
+		setClosed() {
+			this.open = false;
+		},
+		mouseMoveHandler(e: MouseEvent) {
+			const MOUSE_STRAY_DISTANCE = 100;
+
+			// Close the popover if the mouse has strayed far enough from its bounds
+			if (this.isMouseEventOutsidePopover(e, MOUSE_STRAY_DISTANCE)) {
+				this.setClosed();
+			}
```

Comment:
I am not quite convinced that it should automatically close when you move the cursor out of the bounds of the popover.

Reply:
This is Blender's behavior, and I quite like it. It is on my radar as a topic for doing proper UX studies instead of riding with the assumption that Blender's choice was objectively correct. It does perhaps take some getting used to popovers (and dropdown menus) disappearing when your mouse leaves the vicinity, but it also provides a very convenient method of dismissing menus you aren't interacting with. We can also adjust the `MOUSE_STRAY_DISTANCE` from 100px to something different (Blender seems to use about 54 pixels per my counts, but I felt that was a little on the low side). We can also consider leaving more important popovers like the color picker up until a click or the Escape key dismisses them. Let's keep the code for now and study the UX question once we have users.

Reply:
Should we maybe add a specific kind of comment/tag for ux preference stuff?
Basically that should be configurable in the future/impacts ux can be tagged to later easily search for all such occurrences. (Might be good to use a unified approach for both rust and vue)